Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG PY_VERSION=3.7
+FROM continuumio/miniconda3:4.8.2-alpine AS builder
+
+EXPOSE 8888
+
+LABEL maintainer.name="mosdef-hub"\
+      maintainer.url="https://mosdef.org"
+
+ENV PATH /opt/conda/bin:$PATH
+
+USER root
+
+ADD . /mbuild
+
+WORKDIR /mbuild
+
+RUN conda update conda -yq && \
+	conda config --set always_yes yes --set changeps1 no && \
+	conda config --add channels omnia && \
+	conda config --add channels conda-forge && \
+	conda config --add channels mosdef && \
+	conda create -n mbuild-docker python=$PY_VERSION && \
+	. /opt/conda/etc/profile.d/conda.sh && \
+	conda activate mbuild-docker && \
+	conda install python=$PY_VERSION nomkl --file requirements-dev.txt && \
+        python setup.py install && \
```

Comment:
It looks like we're installing python twice?

Reply:
Yeah, for me the default python kept going to 3.8. Manually specifying it here seemed to prevent that behavior.

Reply:
From L22 it initially installs python 3.8.2 and then downgrades to python 3.7.6 in line 25 since the conda version of mbuild doesn't support 3.8 yet.

Reply:
Can we consolidate these two lines into one? Something like 

```sh
conda config --add channels mosdef && \
. /opt/conda/etc/profile.d/conda.sh && \
conda create -n mbuild-docker python=$PY_VERSION nomkl --file requirements-dev.txt && \
conda activate mbuild-docker && \
python setup.py install && \
```